### DISCUSSION TITLE
How to bind "Save Value Interval"

### DISCUSSION THREAD

**Discussion by brijeshvpatel** [frangoteam/FUXA#996] [General]
Refer following image, I want to save "Temprature" at specific interval which will be driven from Tag "cyclicFreq" value. Can someone help to guide to bind "cyclicFreq" tag to "Save Interval Value" field of "Temprature" tag?

Thanks in advance.

![image](https://github.com/frangoteam/FUXA/assets/2650872/6b687d38-3faf-4c12-a52f-088fb02963ff)

**Comment by robbudge** [comment]
I think the save value interval is simply an automatic DB Save if the  tag has not changed.
'Save Value if changed' causes every update  to save  the DB.
I find this  particularly useful for tags that don't change very often. i often have this value set  for 60 seconds. then when producing a graph the tag has a datapoint every 60 seconds even though  it hasn't changed.
active tags have  a datapoint on each update normally 1s interval from the PLC.

**Reply by brijeshvpatel** [reply]
Hi Rob,
Thank you for reply.
I understand your point, but this interval needs to be set dynamically from some button click event or so. Or this can be mapped with some tag so, frequency to fetch and store data in to DB will be changed based on the tag value which can be 1 sec, 60 sec or 15 minutes as per tag value(in ms).

**Reply by unocelli** [reply]
@brijeshvpatel Hi, interesting though I have never seen this kind of setup, can you explain the use case

**Reply by brijeshvpatel** [reply]
Hi Unocelli,

It is very normal to change the data save frequency or showing on to the chart by operator. In very simple usecase, operator wants to see temperature sensor data in HMI at frequency of 100ms, 1 sec, 60 seconds or 1 minute of interval based on their selection. Based on interval selection at the runtime data will be saved in to the DB and chart/datatable visually also show this data automatically at the selected frequency.
for example in Delta HMI, you can select a tag or internal variable to the historybuffer's (for history and realtime data) "interval" option so, when user can change it from HMI visual option or PLC can give tag value to select specific interval to fetch data or show data.

**Comment by robbudge** [comment]
This appears to be more with the way the data is displayed. Not the way data is stored. 
I know with Grafana / Influx you can specify the maximum number of points, this returns a mean point of every 30 minutes.
